src/Components/Common/LoadingOverlay.jsx

```diff
@@ -17,12 +17,12 @@ const OverlayWrapper = styled.div`
   }
 `;
 
-function LoadingOverlay({ spinnerProps }) {
-  //   const radius = spinnerProps.radius ? spinnerProps.radius : 30;
+function LoadingOverlay({ show, spinnerProps }) {
+  if (!show) return null;
   return (
     <OverlayWrapper>
       <div>
-        <Spinner {...spinnerProps} />
+        <Spinner color="#333" radius={64} visible {...spinnerProps} />
```

Comment:
Is there a conflict for radius? It is set to 64 here but one of the spinnerProps above sets radius to 120 and the other sets it to 220. I assume the 64 is a default and gets overridden which is fine but just wanted to confirm!

Reply:
I did a test and it all seems to work fine. The smaller radius is for the dialog

Reply:
Yep 64 is just meant to be the default.